cmd/tdexd/main.go

```diff
@@ -51,13 +51,11 @@ var (
 	tradeSvcPort                  = config.GetInt(config.TradeListeningPortKey)
 	operatorSvcPort               = config.GetInt(config.OperatorListeningPortKey)
 	crawlerIntervalInMilliseconds = time.Duration(config.GetInt(config.CrawlIntervalKey)) * time.Millisecond
-	crawlerLimit                  = config.GetInt(config.CrawlLimitKey)
-	crawlerTokenBurst             = config.GetInt(config.CrawlTokenBurstKey)
 	explorerTimoutRequest         = config.GetDuration(config.ExplorerRequestTimeoutKey)
 	cbMaxFailingRequest           = config.GetInt(config.CBMaxFailingRequestsKey)
 	cbFailingRatio                = config.GetFloat(config.CBFailingRatioKey)
 	rescanRangeStart              = config.GetInt(config.RescanRangeStartKey)
```

Comment:
Semantically is not anymore a range, so if we drop end, we should just call it `RESCAN_START`